Lib/test/test_httpservers.py

```diff
@@ -17,8 +17,11 @@
 import urllib.parse
 import html
 import http.client
+import email.message
```

Comment:
I don't understand. In the current version, the imports are not in alphabetic order : 
```python
from http.server import BaseHTTPRequestHandler, HTTPServer, \
     SimpleHTTPRequestHandler, CGIHTTPRequestHandler
from http import server, HTTPStatus

import os
import sys
import re
import base64
import ntpath
import shutil
import urllib.parse
import html
import http.client
import tempfile
import time
from io import BytesIO

import unittest
from test import support
```
There doesn't seem to be any logical order either - `http.client` is not grouped with `http.server` for instance.

Would you mind explaining more what you want and why ? It will be helpful for next Pull Requests.